sdk/template/Azure.Template/src/Generated/TemplateClient.cs

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+namespace Azure.Template
+{
+    /// <summary> The Template service client. </summary>
+    public partial class TemplateClient
+    {
+        private static readonly string[] AuthorizationScopes = new string[] { "https://api.adu.microsoft.com/.default" };
```

Comment:
Why this auth scope?  Should we provide one that is more clearly an example?

Reply:
use a faked scope https://sample/.default